src/OrchardCore/OrchardCore.OpenId.Core/YesSql/Indexes/OpenIdApplicationIndex.cs

```diff
@@ -10,9 +10,9 @@ public class OpenIdApplicationIndex : MapIndex
         public string ClientId { get; set; }
     }
 
-    public class OpenIdApplicationByPostLogoutRedirectUriIndex : ReduceIndex
+    public class OpenIdAppPostLogoutRedirectIndex : ReduceIndex
```

Comment:
Should we remove `Post` from the class name to match the new property name?

Reply:
Good catch

Reply:
What about `OpenIdApplicationLogoutUriIndex`? Would that be too long?

Reply:
I changed everything to `...App...`